.github/workflows/deploy-happy-stack.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             DOMAIN_NAME=public-frontend.production.single-cell.czi.technology
             ALIAS=cellxgene.cziscience.com
           else
-            DOMAIN_NAME=public-frontend.${DEPLOYMENT_STAGE}.single-cell.czi.technology
+            DOMAIN_NAME=frontend.${DEPLOYMENT_STAGE}.single-cell.czi.technology
```

Comment:
We can keep that change that was previous on line 70, removing the public prefix.

Reply:
nevermind this is temporary and will not be merged back to main. ignore my above comment